src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();
+		this._$method$ = methodName;
 	}
 
 	@Parameter("IFace")
-	CtTypeReference<?> ifaceType;
+	String _IFace;

```

Comment:
So please use

@Parameter
CtTypeReference<?> IFace;

because otherwise it generates not compilable type - note that nested interface is already marked @Local.